Republish for 43 (#1225)
emqtt/emq-docs-cn: en_US/rule/action_republish.md

# Republish Action

The republish action is used to reissue a new MQTT message. The `Topic`, `QoS`, `Retain`, and `Payload` of the new message can be specified by parameters.

> Note that the message republish action does not terminate the delivery of old messages. For example, if a message "t/1" is received by the republish action and then a new "t/2" message is reissued, the delivery of the "t/1" message is not terminated, and clients subscribed to the "t/1" topic can still receive the message.

Reviewer: ```suggestion
Note that the message republish action does not terminate the delivery of the original messages. For example, if a message "t/1" is received by the republish action and then a new "t/2" message is published per this rule, the delivery of the "t/1" message is not terminated, and clients subscribed to the "t/1" topic can still receive the original message.
```

## Create Rule

Click on Rules Engine - Rules - Create, and enter SQL.

```SQL
SELECT

    *

FROM

  "t/1"
```

> The SQL in the document is only for example, please write the actual requirements according to the business.

Reviewer 2: Maybe remove this sentence? I think this is true for any example.
